Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -22,7 +22,8 @@ class DashboardViewModel {
     var sortOption: SortOption = .custom
     var vehicles = [Vehicle]()
     var searchText: String = ""
-    
+    var fetchingMaintenanceEvents = false
```

Comment:
```suggestion
    var isLoading = false
```

I want to make this more generic for the future